cmd/infractl/cluster/list/command.go

```diff
@@ -36,13 +36,15 @@ func Command() *cobra.Command {
 
 	cmd.Flags().Bool("all", false, "include clusters not owned by you")
 	cmd.Flags().Bool("expired", false, "include expired clusters")
+	cmd.Flags().BoolP("quiet", "q", false, "only output cluster names")
```

Comment:
Please add the new flag with an example to https://github.com/stackrox/infra/pull/1328/files#diff-ea79e04ad348f4a92b9b4526f9dd772fe0a5728998c6981b350aaf2c6ca36b83L13-L23